### PR TITLE
GetTime - Extra options

### PR DESCRIPTION
**Pull request by AndyDemisto**
Now supports getting Hour, so you can check if inbetween a certain working hours time

### CODE REVIEW

File: Scripts/script-GetTime.yml [hunk 1]
```diff
@@ -39,6 +39,29 @@ script: |-
           case 'Date':
               timeStr = d.toDateString();
               break;
+        case 'year':
```
Comment (meirwah) [inline Scripts/script-GetTime.yml]: there seems to be indentation between the case lines.. can you fix that?

Comment (anara123) [reply Scripts/script-GetTime.yml]: @AndyDemisto ??

File: Scripts/script-URLDecode [hunk 2]
```diff
@@ -0,0 +1,24 @@
+commonfields:
+  id: a2646164-41d7-4652-8130-94c74cb227fd
```
Comment (anara123) [inline Scripts/script-URLDecode]: change the id to be URLDecode
and version to be -1

**Comment by anara123** [comment]
@AndyDemisto please add test playbook . If you need help with that, please let me know

**Comment by Itay4** [comment]
@AndyDemisto What is the status of this?

**Comment by AndyDemisto** [comment]
Hi Itay,
The script works, but after fighting with Github for an hour my submission had weird spacing as copy pasting it in using a browser changed something.  But I gave up.

**Pull request merged**